subprojects/docs/src/docs/userguide/javaPlugin.xml

```diff
@@ -378,7 +378,10 @@
                 <td>compile</td>
                 <td>-</td>
                 <td>-</td>
-                <td>Compile time dependencies</td>
+                <td>
+                    Compile time dependencies. This configuration is weakly deprecated in favor of the <code>implementation</code> configuration defined
```

Comment:
❌ We should add `implementation`, `testImplementation` and `runtimeOnly` to this list instead of linking to the java-library plugin. These configurations are available in the `java` plugin already. Instead, we can either link back here from `java-library` or just duplicate it in both places.